### ISSUE TITLE
Reimplement import-sql in python

### ISSUE THREAD

**Issue by nyurik** [openmaptiles/openmaptiles-tools#376]
The current [import-sql](https://github.com/openmaptiles/openmaptiles-tools/commit/7655ca25a7a744c8d0106174d53562dd6bf99f09) is implemented using shell script + `xargs` to parallelize file import.

It currently has a number of limitations, mostly dealing with error handling -- if one of the SQL files fail, it is not trivial to find which file failed and where, plus in some cases the script (xargs) can exit without waiting for files that have not yet completed - thus confusing the output even further.

One simple solution would be to rewrite it in Python, allowing more control and helping readability.

See also #373

**Comment by Falke-Design** [comment]
I just want to throw the following into the room: If a script runs on error and then the other scripts are still running until end, it can go, depending upon data size, very long until it is terminated.

**Comment by nyurik** [comment]
I agree, I actually had some parameters in docs for that, but I was too lazy to implement sending SIGINT to all running processes.
